<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectsAdmin</class>
 <widget class="QWidget" name="ProjectsAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font: 24pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>10</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 13pt &quot;Arial&quot;;
 background-color: #2D2D2D; /* Dark grey background */
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="toplabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>5</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 24pt &quot;Arial&quot;;
background-color: #9EB3C0;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Projects&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="createProjectButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 13pt &quot;Arial&quot;;
background-color: #22344C;
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Create Project</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="project_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>1161</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_project">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 13pt &quot;Arial&quot;;
background-color: #22344C;
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Delete Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_project_2">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>370</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 13pt &quot;Arial&quot;;
background-color: rgb(200, 200, 200);
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Delete Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createProjectButton_2">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>370</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 13pt &quot;Arial&quot;;
background-color: rgb(200, 200, 200);
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Create Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton_2">
   <property name="geometry">
    <rect>
     <x>1830</x>
     <y>310</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 13pt &quot;Arial&quot;;
background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editProjectButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 13pt &quot;Arial&quot;;
background-color: #22344C;
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Edit Project</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
